DQM/HGCal/plugins/HGCalTriggerClient.cc

```diff
@@ -239,17 +240,25 @@ void HGCalTriggerClient::analyze(const edm::Event& iEvent, const edm::EventSetup
   bool isEngE0 = false;
   bool isEngE1 = false;
 
-  if (trigType == 0x0001)
+
+  switch(trigType){
+  case Phys:
     h_trigtype->Fill(0);
-  if (trigType == 0x0002)
+    break;
+  case Calib:
     h_trigtype->Fill(1);
-  if (trigType == 0x0004)
+    break;
+  case Random:
     h_trigtype->Fill(2);
-  if (trigType == 0x0008)
+    break;
+  case Soft:
     h_trigtype->Fill(3);
-  if (trigType == 0x0010)
+    break;
+  case Regular:
     h_trigtype->Fill(4);
-
+    break;
```

Comment:
Consider adding a `default` statement. There are several possible [conventions](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-default).

Reply:
The default option has been added for unknown trigger types (please see [def7b37](https://github.com/CMS-HGCAL/cmssw/pull/86/commits/def7b3763f165986b4342110f3f0dc38856f0abf) ).